src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -3862,7 +3862,7 @@ void processResource(IOperation resource, ArrayBuilder<(IVariableDeclarationOper
 
                 LeaveRegion();
 
-                AddDisposingFinally(resource, knownToImplementIDisposable: true, iDisposable, isAsynchronous);
+                AddDisposingFinally(resource, knownToImplementIDisposable: true, iDisposable, disposeMethod, isAsynchronous);
```

Comment:
Well, it's certainly not know to implement `IDisposable` if it's a ref struct. Please update the comments above and the parameter name.